ivy_tests/test_ivy/test_frontends/test_scipy/test_linalg/test_linalg.py

```diff
@@ -403,3 +440,33 @@
 #         on_device=on_device,
 #         a=x[0],
 #     )
+
+# # det
+# @handle_frontend_test(
+#     fn_tree="scipy.linalg.det",
+#     dtype_and_x=_get_dtype_and_matrix(),
```

Comment:
We would probably need to pass in `symmetric=True` here since determinants of asymmetric matrices are undefined